content/docs/setup/behavioral_changes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Behavioral Changes in 1.1
+description: Major changes in behavior, both backward compatibility breaking and non, in Istio 1.1.
+weight: 15
+icon: setup
+
+---
+
+# Major Behavioral Changes in Istio 1.1
+
+## Installer Changes
+
+### Changes with upgrade impact
+
+- Istio’s CRDs have been placed into their own Helm chart `istio-init`.  This prevents loss of custom resource data, facilitates the upgrade process, and enables the Istio project to evolve beyond a Helm-based installation.  The upgrade documentation (**TODO link**) provides the proper procedure for upgrading from Istio 1.0.6 to Istio 1.1.0.  Please follow these instructions **carefully and precisely** when upgrading.  If certmanager is desired, it is mandatory to use the `--set certmanager=true` flag when installing both istio-init and istio charts with either `template` or `tiller` installation modes.
```

Comment:
During the release notes process, I sorted out the links for this section and cleaned up the content significantly.  Feel free to use for inspiration:

https://github.com/istio/istio.io/pull/3704/commits/46fc84b428f97e92ea920f8f4f80d3f80450f9a4

Reply:
@sdake @geeknoid @rcaballeromx looks like there is substantial overlap between the behavioral changes doc and release notes.   How should we decide which goes where?

Reply:
@duderino can you point out some examples?  The final release notes did not include the content in https://github.com/istio/istio.io/commit/46fc84b428f97e92ea920f8f4f80d3f80450f9a4 but6 instead had a link to the upgrade instructions.

There may be some overlap remaining however.  We should favor the release notes to the behavioral changes document for content that is duplicated.

Reply:
Ah my mistake